white_balancer/cfg/white_balancer.cfg

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python
 # configuration
 
-from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, int_t
+from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, int_t, double_t
 
 gen = ParameterGenerator()
 
 gen.add("temp", int_t, 1, "Color temp for white balance.", min=1000, max=12000)
+gen.add("delay", double_t, 1, "Delay the image time stamp by that value.", min=-2.0, max=2.0)
```

Comment:
Delay is the wrong word here. Maybe change the sign?

Reply:
I renamed delay to offset.